github/projects.go

```diff
@@ -267,6 +267,9 @@ type ProjectCard struct {
 	Note       *string    `json:"note,omitempty"`
 	CreatedAt  *Timestamp `json:"created_at,omitempty"`
 	UpdatedAt  *Timestamp `json:"updated_at,omitempty"`
+	Url        *string    `json:"url,omitempty"`
```

Comment:
This field should be `URL` to match Go style, see https://golang.org/s/style#initialisms.